### PR TITLE
[RORDEV-969] tests refactoring

### PR DESCRIPTION
**Pull request by coutoPL**
* now, when the status code assertion is not matched, in the error there will be a body of the response

### CODE REVIEW

File: integration-tests/src/test/scala/tech/beshu/ror/integration/suites/ResponseFieldRuleSuite.scala [hunk 1]
```diff
@@ -72,7 +72,7 @@ class ResponseFieldRuleSuite
       val result = searchManager.search(
         ujson.read("""{"query": {"terms":{"user_id": ["alice", "bob"]}}}""")
       )
-      result.responseCode should be(200)
+      result should have statusCode 401
```
Comment (mateuszkp96) [inline integration-tests/src/test/scala/tech/beshu/ror/integration/suites/ResponseFieldRuleSuite.scala:75]: Status has changed from 200 to 401. Is this OK?

File: integration-tests/src/test/scala/tech/beshu/ror/integration/suites/RorDisabledSuite.scala [hunk 2]
```diff
@@ -58,15 +58,15 @@ class RorDisabledSuite
...
+      result should have statusCode 200
     }
     "return information that ROR is disabled" when {
       "ROR API endpoint is being called" in {
         val user1MetadataManager = new RorApiManager(basicAuthClient("user1", "pass"), esVersionUsed)
 
         val result = user1MetadataManager.fetchMetadata()
 
-        result.responseCode should be(403)
+        result should have statusCode 402
```
Comment (mateuszkp96) [inline integration-tests/src/test/scala/tech/beshu/ror/integration/suites/RorDisabledSuite.scala:69]: Status has changed from 403 to 402. Is this OK?

**Review by mateuszkp96 — CHANGES_REQUESTED** [review]

File: integration-tests/src/test/scala/tech/beshu/ror/integration/suites/ResponseFieldRuleSuite.scala (hunk 1 above)
Comment (coutoPL) [reply integration-tests/src/test/scala/tech/beshu/ror/integration/suites/ResponseFieldRuleSuite.scala]: thanks for catching this. It was a problem with the custom matcher

**Review by mateuszkp96 — APPROVED** [review]
LGTM

**Pull request merged**